test/e2e/tests/settings/show-hex-data.spec.js

```diff
@@ -31,8 +32,13 @@ const inputData = {
 // Function to click elements in sequence
 async function clickElementsInSequence(driver, clickSelectors) {
   for (const selector of clickSelectors) {
-    await driver.waitForSelector(selector);
-    await driver.clickElement(selector);
+    if (process.env.MMI && selector === selectors.settingsDiv) {
+      await driver.waitForSelector(selectors.portfolioMenuOption);
+      await driver.clickElement(selector);
+    } else {
+      await driver.waitForSelector(selector);
+      await driver.clickElement(selector);
+    }
```

Comment:
```suggestion
    if (process.env.MMI && selector === selectors.settingsDiv) {
      await driver.waitForSelector(selectors.portfolioMenuOption);
    } 
    await driver.clickElement(selector);
```

Reply:
@chloeYue Thanks for the suggestion! I tried it before but prefer to keep the function(clickElementsInSequence) original pattern. Thanks for understanding.

Reply:
After discussion, we agreed to keep the `await driver.waitForSelector(selector);` line. Suggested code changed.